vars/common.yml

```diff
@@ -40,7 +40,7 @@ platinum_source: "amzn2"
 
 #Python Settings
 python_bin_dir: "/usr/bin/"
-python_ver: "{{ '3.8' if platinum_source == 'amzn2' else '3.6' }}"
+python_ver: "{{ '3' if platinum_source == 'amzn2' else '3.6' }}"
```

Comment:
Curious about the 3.6 fallback here.

Reply:
fall back will default version in amzn2 and we are keeping as is , with amzn2023 we may not needed as by default comes with 3.11